notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings_palm.ipynb

```diff
@@ -0,0 +1,2028 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    model = TextEmbeddingModel.from_pretrained("textembedding-gecko-001")

Should this be inside the function?
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>No, as the same model is used each time. Including it in the function will just add latency.</p>